server/index.js

```diff
@@ -13,13 +13,14 @@ let templatize = require("./templatize");
 let Request = require("./request");
 let Security = require("./security");
 let localize = require("./localize");
-let errorhandling = require("./lib/errorhandling");
+const HttpError = require("./lib/http-error.js");
 let middleware = require("./lib/middleware")();
 let routes = require("./routes")();
-let getFileList = require("./lib/utils").getFileList;
+const Utils = require("./lib/utils");
 
 let server = express();
-let isDevelopment = env.get("NODE_ENV") === "development";
+const environment = env.get("NODE_ENV");
```

Comment:
should I use that? Yeah it's pretty inconsistent right now. The idea is to do a full es6 re-write once node 6 is lts (october); so this is just to make that patch a bit smaller.